ambry-network/src/main/java/com/github/ambry/network/http2/Http2NetworkClient.java

```diff
@@ -91,6 +95,10 @@ public Http2NetworkClient(Http2ClientMetrics http2ClientMetrics, Http2ClientConf
               http2ClientMetrics.http2StreamAcquireTime.update(System.currentTimeMillis() - streamInitiateTime);
               long streamAcquiredTime = System.currentTimeMillis();
               Channel streamChannel = future.getNow();
+              streamChannel.pipeline().addLast(http2ClientStreamStatsHandler);
+              // TODO: implement ourselves' aggregator. Http2Streams to Response Object
```

Comment:
Just curious: Is there a specific issue this TODO will help with?

Reply:
I can create one.